### PR TITLE
Update hardware-requirements-for-the-teams-app.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7127
Added info

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/hardware-requirements-for-the-teams-app.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ All of the requirements in the following sections apply to both the Microsoft Te
 | Component | Requirement |
 |---------|---------|
 |Computer and processor    | Minimum 1.6 GHz (or higher), 2 core<br><br>Note: For Intel processors, the maximum speed achieved using Intel Turbo Boost Technology (Max Turbo Frequency) must be considered         |
-|Memory     |    4.0 GB RAM     |
+|Memory     |    4.0 GB RAM  (Teams requires a dedicated 4GB of RAM over and above any other system requirements)    |
```
Comment (JohanFreelancer9) [inline Teams/hardware-requirements-for-the-teams-app.md:34]:
```suggestion
|Memory     |    4.0 GB RAM  (Teams requires a dedicated 4 GB of RAM over and above any other system requirements)    |
```

File: Teams/hardware-requirements-for-the-teams-app.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ All of the requirements in the following sections apply to both the Microsoft Te
 | Component | Requirement |
 |---------|---------|
 |Computer and processor    | Intel Core Duo processor |
-|Memory     |   4.0 GB RAM      |
+|Memory     |   4.0 GB RAM  (Teams requires a dedicated 4GB of RAM over and above any other system requirements)     |
```
Comment (JohanFreelancer9) [inline Teams/hardware-requirements-for-the-teams-app.md:50]:
```suggestion
|Memory     |   4.0 GB RAM  (Teams requires a dedicated 4 GB of RAM over and above any other system requirements)     |
```

File: Teams/hardware-requirements-for-the-teams-app.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ All of the requirements in the following sections apply to both the Microsoft Te
 | Component | Requirement |
 |---------|---------|
 |Computer and processor    | 1.6 GHz (or higher) (32-bit or 64-bit), 2 core        |
-|Memory     |    4.0 GB RAM     |
+|Memory     |    4.0 GB RAM  (Teams requires a dedicated 4GB of RAM over and above any other system requirements)   |
```
Comment (JohanFreelancer9) [inline Teams/hardware-requirements-for-the-teams-app.md:64]:
```suggestion
|Memory     |    4.0 GB RAM  (Teams requires a dedicated 4 GB of RAM over and above any other system requirements)   |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @lindspea - could you please commit my three suggested changes? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by lindspea** [comment]
@SerdarSoysal

**Review by cazawideh — APPROVED** [review]

**Pull request merged**